docs/source/examples/managed-jobs.rst

```diff
@@ -452,6 +452,286 @@ To submit the pipeline, the same command :code:`sky jobs launch` is used. The pi
+Workers in the pool are **reused** across job submissions, avoiding repeated setup and **saving cold start time**. This is ideal for workloads where many jobs need to run with the same software environment and dependencies.
+
+
+.. tip::
+
+  To get started with pools, use the nightly build of SkyPilot: ``pip install -U skypilot-nightly``
+
+Create a pool
+~~~~~~~~~~~~~
+
+Here is a simple example of creating a pool:
+
+.. code-block:: yaml
+  :emphasize-lines: 2-4
+
+  # pool.yaml
+  pool:
+    # Specify the number of workers in the pool.
+    workers: 3
+
+  resources:
+    # Specify the resources for each worker, e.g.
+    accelerators: {H100:1, H200:1}
+
+  file_mounts:
+    /my-data:
+      source: s3://my-dataset/
+      mode: MOUNT
+
+  setup: |
+    # Setup commands for all workers
+    echo "Setup complete!"
+
+Notice that the :code:`pool` section is the only difference from a normal SkyPilot YAML.  
+To specify the number of workers in the pool, use the :code:`workers` field under :code:`pool`.  
+When creating a pool, the :code:`run` section is ignored.
+
+
+To create a pool, use the following command:
+
+.. code-block:: console
+
+  $ sky jobs pool apply --pool gpu-pool pool.yaml
+  YAML to run: pool.yaml
+  Pool spec:
+  Worker policy:  Fixed-size (3 workers)
+
+  Each pool worker will use the following resources (estimated):
+  Considered resources (1 node):
+  -------------------------------------------------------------------------------------------------------
+  INFRA                 INSTANCE                         vCPUs   Mem(GB)   GPUS     COST ($)   CHOSEN   
+  -------------------------------------------------------------------------------------------------------
+  Nebius (eu-north1)    gpu-h100-sxm_1gpu-16vcpu-200gb   16      200       H100:1   2.95          ✔     
+  Nebius (eu-north1)    gpu-h200-sxm_1gpu-16vcpu-200gb   16      200       H200:1   3.50                
+  GCP (us-central1-a)   a3-highgpu-1g                    26      234       H100:1   5.38                
+  -------------------------------------------------------------------------------------------------------
+  Applying config to pool 'gpu-pool'. Proceed? [Y/n]: 
+  Launching controller for 'gpu-pool'...
+  ...
+  ⚙︎ Job submitted, ID: 1
+
+  Pool name: gpu-pool
+  📋 Useful Commands
+  ├── To submit jobs to the pool: sky jobs launch --pool gpu-pool <yaml_file>
+  ├── To submit multiple jobs:    sky jobs launch --pool gpu-pool --num-jobs 10 <yaml_file>
+  ├── To check the pool status:   sky jobs pool status gpu-pool
+  └── To terminate the pool:      sky jobs pool down gpu-pool
+
+  ✓ Successfully created pool 'gpu-pool'.
+
+The pool will be created in the background. You can submit jobs to this pool immediately; jobs will remain in the PENDING state until the worker cluster is ready, and will start automatically once workers are available.
```

Comment:
"jobs will remain in the PENDING state until the worker cluster is ready, and will start automatically once workers are available." a bit confusing. Is there a diff between ready and available?

Reply:
```
The pool will be created in the background. You can submit jobs to this pool immediately. If there aren't any workers ready to run the jobs yet, the jobs will wait in the PENDING state.
The jobs will start automatically once workers are provisioned and ready to run.
```
Seem ok?

Reply:
"Jobs will start automatically once some worker is provisioned and ready to run." To be clearer than we just need some worker, not all.